website/content/docs/connect/config-entries/sameness-group.mdx

```diff
@@ -6,16 +6,10 @@ description: |-
 
 # Sameness groups configuration entry reference
 
-This page provides reference information for sameness group configuration entries. Sameness groups associate services with identical names across partitions and cluster peers.
+This page provides reference information for sameness group configuration entries. Sameness groups associate identical admin partitions to facilitate traffic between identical serices registered to different datacenters.
```

Comment:
```suggestion
This page provides reference information for sameness group configuration entries. Sameness groups associate identical admin partitions to facilitate traffic between identical services registered to different datacenters.
```

Similar to the sentiment that Michael pointed out, I feel like the concept that sameness groups can be a mixture of local partitions (same dc) and remote partitions (peered dc) is not conveyed clearly everywhere and might be confusing for users if not mentioned up-front in an explicit way.